src/ElementUtilities.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Returns closest ancestor of element which matches selector.
+ * Similar to 'closest' methods as seen here:
+ *  https://api.jquery.com/closest/
+ *  https://developer.mozilla.org/en-US/docs/Web/API/Element/closest
+ * @param  {!HTMLElement} el    Element
```

Comment:
Feel free to ignore this: this function appears to only actually only require [Element](https://developer.mozilla.org/en-US/docs/Web/API/Element), not [HTMLElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement) specifically. For our code, I think the only time we'll generally require an HTMLElement is when using HTMLElement.style or HTMLElement.click() and then prefer Element or [Node](https://developer.mozilla.org/en-US/docs/Web/API/Node).